helm/prometheus-rules/templates/platform/atlas/alerting-rules/loki.rules.yml

```diff
@@ -83,3 +83,28 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    - alert: LokiNeedsToBeScaledDown
+      annotations:
+        description: '{{`Loki component {{ $labels.labelpod }} is consuming very few resources and needs to be scaled down.`}}'
```

Comment:
:bulb: Would there be a way to say which resources are under-used (CPU or RAM) in the description?

Reply:
Well, the query uses `and` so it should be both memory and CPU being underused that should trigger the alert. 

However I did consider your point in case you'd prefer using `or` instead of `and` and tried the following query :
```
sum by (cluster_id, installation, namespace, pipeline, provider, labelpod, unit) (label_replace((label_replace(container_memory_working_set_bytes{container="loki", namespace="loki"}, "labelpod", "$1", "pod", "(loki-[[:alnum:]]*)-.*")) , "unit", "byte", "", ""))
          / 
        sum by(cluster_id, installation, namespace, pipeline, provider, labelpod, unit) (label_replace(kube_pod_container_resource_requests{container="loki", namespace="loki", unit="byte"}, "labelpod", "$1", "pod", "(loki-[[:alnum:]]*)-.*")) 
          <= 0.30
        and
        sum(label_replace((label_replace(rate(container_cpu_usage_seconds_total{container="loki", namespace="loki"}[5m]), "labelpod", "$1", "pod", "(loki-[[:alnum:]]*)-.*")), "unit", "core", "", "")) by (cluster_id, installation, namespace, pipeline, provider, labelpod, unit) 
          / 
        sum by(cluster_id, installation, namespace, pipeline, provider, labelpod, unit) (label_replace(kube_pod_container_resource_requests{container="loki", namespace="loki", unit="core"}, "labelpod", "$1", "pod", "(loki-[[:alnum:]]*)-.*")) 
          <= 0.30
```
But this leads to a conflict between the 2 "unit" labels (one being `byte`, the other `core`) and thus no data is displayed and it can't be used in the description.

Moreover, when both resources are underusesed, it becomes harder to display those in the description.